packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/styles.tsx

```diff
@@ -58,3 +62,35 @@ export const ButtonWrapper = styled.div`
   width: 100%;
   flex-direction: column;
 `
+
+export const ListWrapper = styled.div`
+  max-width: 356px;
```

Comment:
does this cause issues on mobile @milan-bc ?

Reply:
Nope it works as a charm :)